docs/FAQ.md

```diff
@@ -226,6 +235,19 @@ For connection to complete, your contact has to be online and have the app runni
 
 Once the connection is established you don't need to be online at the same time to send messages.
 
+### I do not know my database passphrase
+
+If you are prompted to enter database passphrase and you do not know it, this could have happened due to:
+- Error during SimpleX Chat installation on desktop.
+- Migration of app data from one device to another while using unsupported migration tool (Use SimpleX Chat's own migration tool only)
+- You may have forgotten the passphrase. (There is no other way to access your data)
```

Comment:
```suggestion
- You may have forgotten the passphrase. (There is no other way to access your data).
- Migration of app data from one device to another while using unsupported migration process, e.g. via iCloud backup. Use SimpleX Chat's own migration process in the app Settings.

In the previous desktop app versions it could also happen in case of error during SimpleX Chat installation.
```